CHANGELOG.md

```diff
@@ -3197,7 +3256,7 @@ We'd like to offer a big thanks to the 12 contributors who made this release pos
 - [DataGrid] Remove the `GridEvents` enum (#6003) @flaviendelangle
 - [DataGrid] Remove the deprecated `hide` column property (#5999) @flaviendelangle
 
-### `@mui/x-date-pickers@v6.0.0-alpha.0` / `@mui/x-date-pickers-pro@v6.0.0-alpha.0`
+### `@mui/x-date-pickers@6.0.0-alpha.0` / `@mui/x-date-pickers-pro@6.0.0-alpha.0`
```

Comment:
This was wrong, it now matches what the core is doing. Other examples:

- https://bundlephobia.com/package/@mui/x-date-pickers@6.9.1
- https://unpkg.com/browse/@formkit/pro@0.112.0/